addons/sitting/XEH_clientInit.sqf

```diff
@@ -7,6 +7,6 @@ if !(hasInterface) exitWith {};
 ["isNotSitting", {!((_this select 0) getVariable [QGVAR(isSitting), false])}] call EFUNC(common,addCanInteractWithCondition);
 
 // Handle interruptions
-["medical_onUnconscious", {_this call DFUNC(handleInterrupt)}] call EFUNC(common,addEventhandler);
-["SetHandcuffed", {_this call DFUNC(handleInterrupt)}] call EFUNC(common,addEventhandler);
-["SetSurrendered", {_this call DFUNC(handleInterrupt)}] call EFUNC(common,addEventhandler);
+["medical_onUnconscious", {_player call DFUNC(handleInterrupt)}] call EFUNC(common,addEventhandler);
```

Comment:
Throwing:
 ```
Error position: <_player call ace_sitting_fnc_handleInter>
Error Undefined variable in expression: _player
File z\ace\addons\sitting\XEH_clientInit.sqf, line 10
```

Test codes:
```
[] spawn {
    [player, true] call ace_medical_fnc_setUnconscious;
    sleep 10;
    [player, false] call ace_medical_fnc_setUnconscious;
};

[] spawn {
    ["SetHandcuffed", [player, true]] call ace_common_fnc_localEvent;
    sleep 10;
    ["SetHandcuffed", [player, false]] call ace_common_fnc_localEvent;
};
```


I don't think this should be changed at all and keep it as `_this`.  
For all 3 events the unit is the first argument, which is what handleInterrupt is expecting.